File: 3223-minimum-length-of-string-after-operations/main.py
class Solution:
    def minimumLength(self, s: str) -> int:
        freq = Counter(s)

        del_count = 0
        for frequency in freq.values():
            if frequency & 1: 
                del_count += frequency - 1
            else:
                del_count += frequency - 2
            
        return len(s) - del_count

File: 3223-minimum-length-of-string-after-operations/main.go
func minimumLength(s string) int {

    freq := make(map[rune]int, len(s))
    for _, chr := range s {
        freq[chr] += 1
    }

    deleteCount := 0
    for _, frequency := range freq {
        if frequency & 1 != 0 {
            deleteCount += frequency - 1
        }else {
            deleteCount += frequency - 2
        }
    }

    return len(s) - deleteCount

}
